packages/marketplace/src/types/spicetify.d.ts

```diff
@@ -1371,9 +1371,13 @@ declare namespace Spicetify {
      */
     namespace Config {
         const version: string;
-        const current_theme: string;
-        const color_scheme: string;
-        const extensions: string[];
+        let current_theme: string;
+        let color_scheme: string;
+        let extensions: string[];
         const custom_apps: string[];
+
+        // These two are just added by Marketplace so we can save and restore them when removing a theme
+        let local_theme: string;
+        let local_color_scheme: string;
```

Comment:
I think we can change this in `spicetify-cli` as well right? Might as well 🤷

Reply:
We could but it's meant to be a copy, not a way to modify `config-xpui` from within the app. The only edge case where we would only need to do this would be Marketplace tbh.

Reply:
Does it make sense to save `local_theme` in cli? Or not really since `current_theme` is meant to be local?

Reply:
The latter, yeah. Unless we do want to make Marketplace a part of CLI, there aren't any reason to have both properties have the same value since currently it takes stuff from `config-xpui`